webdriver/tests/bidi/browsing_context/print/margin.py

```diff
@@ -105,14 +107,19 @@ async def test_margin_same_as_page_dimension(
     await bidi_session.browsing_context.navigate(
         context=top_context["context"], url=page, wait="complete"
     )
-    print_value = await bidi_session.browsing_context.print(
-        context=top_context["context"],
-        shrink_to_fit=False,
-        margin=margin,
-    )
 
-    # Check that content is out of page dimensions.
-    await assert_pdf_content(print_value, [{"type": "string", "value": ""}])
+    try:
+        print_value = await bidi_session.browsing_context.print(
+            context=top_context["context"],
+            shrink_to_fit=False,
+            margin=margin,
+        )
+
+        # Check that content is out of page dimensions.
+        await assert_pdf_content(print_value, [{"type": "string", "value": ""}])
+    except UnsupportedOperationException:
+        # Empty content area: https://github.com/w3c/webdriver-bidi/issues/473
+        ...
```

Comment:
nit: seems like we usually use `pass` in this repository
```suggestion
        pass
```